### DISCUSSION TITLE
Tider og tjek ind i klub

### DISCUSSION THREAD

**Discussion by sunebuhl** [scaarup/aula#106] [Q&A]
Jeg har brugt integrationen noget tid med stor success.  Tak for det store arbejde.

Nu er den store skiftet fra BFO / SFO til klub. Her benytter de dog stadig check ind / check ud i Aula og jeg kan se hans status samt tider direkte på aula.dk

Men oplysningerne ligger tilsyneladende ikke længere under den eksisterende entity fra skolen, her findes nu kun ugeplanen.

Jeg har forsøgt at re-loade integrationen, men der kommer ikke nogle nye entities / sensors ind. 

Er der en måde, at få klub integrationen til at virke? 

Her er hvad jeg ser under komme/gå modulet i Aula.

![Skærmbillede 2023-05-30 142356](https://github.com/scaarup/aula/assets/37135644/a6740171-6fe7-45ae-b2f9-86ad421ed5d9)

**Comment by scaarup** [comment]
Hejsa.
Det lyder lidt underligt. Jeg har selv unger, der går i SFO og som bruger "komme/gå" modulet. 
Skal lige forstå det korrekt - både før og efter skiftet til klub, er der anvendt "komme/gå" modulet?

Det er nok bedst at få noget debug log slået til og paste det her.

**Comment by sunebuhl** [comment]
Ja, det er korrekt. Både før og efter skiftet til klub er der anvendt komme/gå modulet. 

Jeg har slået debug logging til og reloadet integrationen. 

Her er lidt der måske er brugbart (kom i forbindelse med reload):

```
Logger: homeassistant.config_entries
Source: helpers/entity_component.py:179
First occurred: 11:27:23 AM (2 occurrences)
Last logged: 11:27:23 AM

Error setting up entry Aula for calendar
Error setting up entry Aula for binary_sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 387, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/usr/src/homeassistant/homeassistant/components/calendar/__init__.py", line 285, in async_setup_entry
    return await component.async_setup_entry(entry)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 179, in async_setup_entry
    raise ValueError("Config entry has already been setup!")
ValueError: Config entry has already been setup!
```




```
Logger: custom_components.aula.client
Source: custom_components/aula/client.py:178
Integration: Aula ([documentation](https://github.com/scaarup/aula/blob/main/README.md), [issues](https://github.com/scaarup/aula/issues))
First occurred: May 30, 2023 at 5:10:11 PM (789 occurrences)
Last logged: 11:27:31 AM

Unable to retrieve presence data from Aula from child with id XXXXXXX. Some data will be missing from sensor entities.
```

**Comment by scaarup** [comment]
Det kan være du kan smide denne linje ind i client.py over for-loopet på linje 172:
`_LOGGER.warn(self._children)`
...og se hvad den spytter ud efter en ha restart.
